### DISCUSSION TITLE
Validation not working in the document example

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#5009] [General]
I´m not sure you saw this @stsrki so adding it here [validation not working](https://github.com/Megabit/Blazorise/issues/4740#issuecomment-1722460009) but your example "[Addon with validation](https://blazorise.com/docs/components/addon)" is not showing the "Enter valid name!" message below as I beleve it should.

<img width="833" alt="image" src="https://github.com/Megabit/Blazorise/assets/2386572/e30975ea-a51c-41e2-a544-3c355d937f74">

I have also not been able to get this to work by moving things around...

**Comment by stsrki** [comment]
We already had a similar report regarding the ValidationError, #4992. I still need to start working on it to confirm.

**Reply by sturlath** [reply]
OK cool.. didn´t see it sorry.
